scanner/BUILD_IMAGE_VERSION

```diff
@@ -0,0 +1 @@
+scanner-build-0.3.59
```

Comment:
I think we could use something far more generic because scanner should be statically compiled. Perhaps we could just use something like docker.io/library/golang:$EXPECTED_GO_VERSION (not you'll have to chop of the go prefix in that file)